setzer/dialogs/include_bibtex_file/include_bibtex_file.py

```diff
@@ -92,7 +92,7 @@ def init_current_values(self):
     def setup(self):
         file_filter1 = Gtk.FileFilter()
         file_filter1.add_pattern('*.bib')
-        file_filter1.set_name('BibTeX files')
+        file_filter1.set_name(_('BibTeX files')) #?
```

Comment:
I'm not entirely sure weather this is necessary.

Reply:
I think yes, but can you check with a different app if they do it?

Reply:
Please mark it as translatable.